src/types.ts

```diff
@@ -583,6 +583,9 @@ export interface DraxContextValue {
 /** Optional props that can be passed to a DraxProvider to modify its behavior */
 export interface DraxProviderProps {
 	debug?: boolean;
+
+	/** Drax parent view style */
+	style: ViewStyle,
```

Comment:
I can't take a look at this in detail right now, but this syntax looks off (comma instead of semicolon), maybe wrap it in a `StyleProp`, and the prop should be made optional:

```
style?: StyleProp<ViewStyle>;
```

Reply:
Sorry about that, mistake from my side. I did actually notice it immediately and pushed an amend, apparently I just forgot to add the changes first. I added the StyleProp wrapping whilst fixing that mistake!

Reply:
As soon as I have an opportunity to do a simple test verification of this locally, I will see about merging and releasing this change. Thank you!